packages/site-settings/src/wizards/WizardHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { WizardHeader as WmeWizardHeader, Logo, ExitButton } from '@moderntribe/wme-ui';
+import { useLocation } from 'react-router-dom';
+import { __ } from '@wordpress/i18n';
+import { useWizard } from '@site/hooks';
+import { StoreBuilderLogo } from '@site/logos';
+import { Box } from '@mui/material';
+
+const EXIT_TEXT = __('Exit to Setup', 'moderntribe-sitebuilder');
+export interface WizardHeaderInterface {
+  hideExit?: boolean;
+}
+
+const WizardHeader: React.FC<WizardHeaderInterface> = () => {
```

Comment:
It's probably a good idea to check out the updated FTC wizard and copy the way it's doing the layout. That way we move forward with a single pattern that will be easy to work with in the future.